turbinia/turbiniactl.py

```diff
@@ -425,7 +425,9 @@ def main():
       sys.exit(1)
   #sends test notification
   if args.command == 'testnotify':
-    notify.sendmail(config.EMAIL_ADDRESS ,'Turbinia test notification', 'This is a test noticiation')
+    notify.sendmail(
+        config.EMAIL_ADDRESS, ' s/noticiation/notification/.',
+        ' s/noticiation/notification/.')
```

Comment:
Oops, sorry for the confusion here.  I just meant that the string "noticiation" should be replaced by "notification", not that the string should literally change to this.